File: go-sdk/pkg/models/operations/getarticlebyid.go
package operations

type GetArticleByIDPathParams struct {
	ID int64 `pathParam:"style=simple,explode=false,name=id"`
}

type GetArticleByIDRequest struct {
	PathParams GetArticleByIDPathParams
}

type GetArticleByIDResponse struct {
	ContentType                            string
	StatusCode                             int64
	GetArticleByID200ApplicationJSONObject map[string]interface{}
}

File: ts-sdk/src/sdk/models/operations/getarticlebyid.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class GetArticleByIdPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=id" })
  id: number;
}


export class GetArticleByIdRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetArticleByIdPathParams;
}


export class GetArticleByIdResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  getArticleById200ApplicationJSONObject?: Record<string, any>;
}
